polylogue/index_qdrant.py

```
@@ -0,0 +1,64 @@
+    api_key = os.environ.get("POLYLOGUE_QDRANT_API_KEY")
+
+    try:  # pragma: no cover - optional dependency
+        from qdrant_client import QdrantClient
+        from qdrant_client.http import models as rest
+    except Exception:
+        return
+
+    client = QdrantClient(url=url, api_key=api_key)
+    try:
+        client.get_collection(collection)
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)